src/routes/userRoute.js

```diff
@@ -3,7 +3,8 @@ const {
   adminBlockUser,
   adminUnblockUser,
   autoFollowUserAnon,
-  setAllFollowerCount
+  setAllFollowerCount,
+  setWPUUser
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The addition of `setWPUUser` from the services seems to be fine. It's imported correctly and doesn't seem to have any issues.
```diff
+  setWPUUser
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->